src/CommandingOfficers/Qis.java

```diff
@@ -39,7 +39,8 @@ public instantiator()
           "What is known is that Qis has become incredibly adept at using the ever-present snow to his advantage."));
       infoPages.add(new InfoPage(
           "Passive:\r\n" +
-          "Watery terrain slows your non-naval units, but all units have perfect movement in snow.\n" +
+          "Watery terrain slows your non-naval vehicles, and they lose -1 movement when starting in rain\n" +
```

Comment:
"lose -1 movement" is awkward phrasing. They gain movement?

Reply:
It is awkward phrasing. I guess I probably picked up using double negatives like that from the AWBW customization chat.

Reply:
It would also be more consistent to rephrase to put the numbers up front; most/all of the other COs do that, I think.
e.g. "-1 movement for non-naval vehicles when moving through watery terrain" or whatever.

Reply:
I can't think of a way to phrase it concisely and clearly, and I think those virtues probably trump consistency?

Reply:
Ah, fair enough I guess. I'm not sure we are perfectly consistent otherwise anyway.